runtime-modules/project-token/src/tests/split.rs

```diff
@@ -296,7 +301,7 @@ fn finalize_split_ok_with_event_deposit() {
         last_event_eq!(RawEvent::RevenueSplitFinalized(
             1u64,
             member!(1).1,
-            DEFAULT_SPLIT_ALLOCATION
+            DEFAULT_SPLIT_ALLOCATION_RATE * DEFAULT_SPLIT_ALLOCATION,
```

Comment:
The term "split allocation" now became kind of ambiguous, because it's used both to mean the channel balance (minus existential deposit) at the time of the split issuance (the `full_amount`) and the amount actually to be split between token holders (`full_amount * split_allocation_rate`). I think we should come up with some 2 separate terms for those and use them consistently.

Reply:
I used `allocation` for the actual gross allocation and `revenue` for `allocation * rate`

Reply:
38a7c6e3a1f0cbb2df2e044be22357bea656e207

Reply:
1. I think `revenue` for gross amount and `allocation` for `revenue * rate` actually makes more sense. My reasoning here being: `revenue` is 100% of the channel revenue, to be split between creator and token holders, `allocation` is the amount that's allocated to token holders only.
2. Make sure to change it in the tests/comments/mocks/traits etc. also

Reply:
fixed here : e6d8a3004abc21147f33f3080a196b8db571ed44